metricbeat/mb/testing/testdata.go

```diff
@@ -42,7 +42,7 @@ import (
 
 const (
 	expectedExtension  = "-expected.json"
-	applicationJson    = "application/json"
+	textPlain          = "text/plain"
```

Comment:
Not a big deal because it's a test, but why was the content type changed?

Reply:
https://github.com/elastic/beats/pull/42748#issuecomment-2666144789 looks like it explains this